src/styles/_utils.scss

```diff
@@ -317,6 +317,18 @@ $sides: (top, right, bottom, left);
   place-self: flex-end !important;
 }
 
+/*  Position
+========================================================================== */
+.p-relative {
+  position: relative !important;
+}
+.p-absolute {
+  position: absolute !important;
+}
+.p-sticky {
+  position: sticky !important;
+}
+
```

Comment:
I don't think we want to keep adding additional utility classes. We are trying to move away from these in the future. Can you just add the styles in the component?